opa/opa_time.c

```diff
@@ -263,7 +263,7 @@ static uint32_t hfi_timebase_from_cpuinfo(uint32_t old_pico_per_cycle)
 #endif
 
 	/* If there's no change (within a small range), just return the old one */
-	if (abs(new_pico_per_cycle - old_pico_per_cycle) < 5)
+	if (abs((int32_t)(new_pico_per_cycle - old_pico_per_cycle)) < 5)
```

Comment:
@aravindksg Is this OK for you?

Reply:
This works fine. But I feel the `max(new, old) - min(new, old)` was cleaner. Could you please do that? (While at it, please assign result of max(), min() operations to temporary variables to avoid having line wraps)